Makefile

```diff
@@ -146,13 +144,14 @@ endif
 .PHONY: bundle
 bundle: manifests kustomize
 	operator-sdk generate kustomize manifests -q
-	cd config/olm && $(KUSTOMIZE) edit set image "quay.io/dynatrace/dynatrace-operator:snapshot"=$(IMG)
-	$(KUSTOMIZE) build config/olm | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+	cd config/olm/$(PLATFORM) && $(KUSTOMIZE) edit set image "docker.io/dynatrace/dynatrace-operator:snapshot"="$(IMG)"
```

Comment:
This is supposed to replace the Deployment image, which has quay.io, so I doubt it's docker.io here.

Reply:
Changed